src/Groups/abelian_aut.jl

```diff
@@ -81,6 +81,7 @@ function (aut::AutGrpAbTor)(f::Union{GrpAbFinGenMap,TorQuadModMor};check::Bool=t
   gene = GAP.Globals.GeneratorsOfGroup(AA)
   img = GAP.Obj([img_gap(a) for a in gene])
   fgap = GAP.Globals.GroupHomomorphismByImagesNC(AA,AA,img)
+  @req fgap in aut.X "Map does not define an element of the group"
```

Comment:
```suggestion
  check || fgap in aut.X || error("Map does not define an element of the group")
```

Reply:
Ah, I confused && and ||

Reply:
`@req check && fgap in aut.X "..."`?

Reply:
Is it a preferable choice or a matter of taste ? Right now it is in line with the first input check of the function. Would you suggest to change both of them ?

Reply:
`@req` will throw an `ArgumentError` and not a generic `ErrorException`. It is used only for argument checking. I guess it is not really appropriate at in this line. Would be for the first line though.